apps/admin/app/(auth)/login/_components/form.tsx

```diff
@@ -0,0 +1,104 @@
+"use client";
+
+import { AuthPage, AuthProviderButton } from "@repo/ui/components";
+import { formSchema, TFormSchema } from "./form-schema";
+import { useForm } from "react-hook-form";
+import { ReactForm } from "@repo/ui/forms";
+import { zodResolver } from "@hookform/resolvers/zod";
+import {
+  Button,
+  Checkbox,
+  CheckboxContainer,
+  Input,
+  Label,
+  Linktag,
+} from "@repo/ui/partials";
+import Link from "next/link";
+
+export function LoginForm() {
+  const form = useForm<TFormSchema>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+  });
+
+  const submitfunction = () => {
+    console.log("Submitted");
```

Comment:
```suggestion
    console.log("Submitted");
```
please remove console